code/game/machinery/computer/communications.dm

```diff
@@ -41,6 +41,12 @@
 	/// The last lines used for changing the status display
 	var/static/last_status_display
 
+	var/toggle_uses = 0 //how many uses the console has done of toggling the emergency access
+	var/toggle_cooldown = 3 //the minimum cooldown of toggling the emergency access IN SECONDS
+	var/toggle_max_uses = 6 //how many uses can you toggle emergency access with before cooldowns start occuring BOTH ENABLE/DISABLE
+	var/time_last_toggled //when emergency access was last toggled
+	var/time_toggle_reset = 30 //how long one must wait without toggling to reset toggle_uses IN SECONDS
```

Comment:
For all of these move the comment above it and have it be ```/// description``` this is for autodoc purposes